src/routes/resource-edit.js

```diff
@@ -110,10 +109,7 @@ class ResourceEditRoute extends Component {
         url: customUrl,
         visibilityData: { isHidden: !isVisible },
         coverageStatement,
-        customEmbargoPeriod: {
-          embargoValue: customEmbargoValue,
-          embargoUnit: customEmbargoUnit
-        },
+        customEmbargoPeriod: customEmbargoPeriod[0] || { embargoValue: 0 },
```

Comment:
please move the condition to a separate method

Reply:
I don't know what the method would look like and how it would help. I think it's better to just save the `{ embargoValue: 0 }` to a variable and then have:
```js
customEmbargoPeriod: customEmbargoPeriod[0] || defaultEmbargoPeriod,
```